tool/Cargo.toml

```diff
@@ -31,3 +31,5 @@ askama = "0.12"
 
 [dev-dependencies]
 insta = { version = "1.7.1", features = [ "backtrace" ] }
+quote = "1.0"
+proc-macro2 = "1.0.79"
```

Comment:
I like using tests to debug and develop the code gen and it helps to have these crates. Happy to remove them once I have the full functionality or before a possible merge, but I would like to keep them while iterative development is happening.